From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Wed, 25 Mar 2020 09:44:39 -0700
Subject: [PATCH] Do not disable unavailable plugin

Latest DNF complains about the `spacewalk` plugin being unavailable when
called by mock. Since it never was available in Clear, silence the
warning.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 mock/py/mockbuild/util.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mock/py/mockbuild/util.py b/mock/py/mockbuild/util.py
index 29a7e33..0d7a8fe 100644
--- a/mock/py/mockbuild/util.py
+++ b/mock/py/mockbuild/util.py
@@ -1156,7 +1156,7 @@ def setup_default_config_opts(unprivUid, version, pkgpythondir, unprivGid=None):
     config_opts['microdnf_common_opts'] = []
     config_opts['rpm_command'] = '/bin/rpm'
     config_opts['rpmbuild_command'] = '/usr/bin/rpmbuild'
-    config_opts['dnf_disable_plugins'] = ['local', 'spacewalk']
+    config_opts['dnf_disable_plugins'] = ['local']
 
     config_opts['opstimeout'] = 0
 
